R/makephyloseqFromTreeSummarizedExperiment.R

```diff
@@ -80,10 +80,10 @@ setMethod("makePhyloseqFromTreeSummarizedExperiment",
         args[["otu_table"]] <- otu_table
 
         # If rowData includes information
-        if(!( length(rowData(x)[,taxonomyRanks(x)]) == 0 ||
-              is.null((rowData(x)[,taxonomyRanks(x)])) )){
+        if(!( length(rowData(x)) == 0 ||
+              is.null((rowData(x))) )){
             # Gets the taxonomic data from rowData, and converts it to tax_table
-            tax_table <- as.matrix(rowData(x)[,taxonomyRanks(x)])
+            tax_table <- as.matrix(rowData(x))
```

Comment:
I think phyloseq's taxonomy table is not that flexible, but still I think that we shouldn't exclude anything

Reply:
you have to because, a matrix can hold only one type. If you have characters and numbers in the `rowData` it will be converted to character anyway and you cannot use it as taxonomic information.

The subsetting should be used, because you don't expect anything else from a phyloseq object

Reply:
Ok, in one data I had some additional information which was not ranks, so I thought that this does too much, I will revert this change